profile-widget/diveeventitem.cpp

```diff
@@ -81,19 +81,19 @@ void DiveEventItem::setupPixmap()
 #define EVENT_PIXMAP(PIX) QPixmap(QString(PIX)).scaled(sz_pix, sz_pix, Qt::KeepAspectRatio, Qt::SmoothTransformation)
 #define EVENT_PIXMAP_BIGGER(PIX) QPixmap(QString(PIX)).scaled(sz_bigger, sz_bigger, Qt::KeepAspectRatio, Qt::SmoothTransformation)
 	if (same_string(internalEvent->name, "")) {
-		setPixmap(EVENT_PIXMAP(":warning-icon"));
+		setPixmap(EVENT_PIXMAP(":status-warning-icon"));
 	} else if (internalEvent->type == SAMPLE_EVENT_BOOKMARK) {
-		setPixmap(EVENT_PIXMAP(":flag"));
+		setPixmap(EVENT_PIXMAP(":bookmark-icon"));
 	} else if (event_is_gaschange(internalEvent)) {
 		struct gasmix *mix = get_gasmix_from_event(&displayed_dive, internalEvent);
 		if (mix->he.permille)
-			setPixmap(EVENT_PIXMAP_BIGGER(":gaschangeTrimix"));
+			setPixmap(EVENT_PIXMAP_BIGGER(":gaschange-tmx-icon"));
```

Comment:
tmx for trimix? I'm not familiar with that abbreviation. Let's use trimix unless there's a good reason for tmx

Reply:
Sure, no problem. I used it, because current icon file-name is *Tmx_change.png* and it looks similar to *air*, *ean*, *oxy* used for the rest gaschage events.

Reply:
I didn't even notice that. I think as we clean it up, the switch to trimix might be easier to understand.